pkg/monitor/monitor.go

```diff
@@ -29,14 +30,15 @@ const (
 	nodeWaitFactor       = 2
 	nodeWaitSteps        = 5
 
+	CSIPodNamePrefix           = "px-csi-ext"
```

Comment:
We have tried to not mention PX specific naming in the common stork code. I would suggest adding a new API in the Driver interface - `GetCSIPodPrefix` and return `px-csi-ext` from Portworx driver implementation. For other drivers you can return an empty value and handle that in monitor and extender code